src/inputs/Table/components/TableToolbar/TableToolbar.js

```diff
@@ -26,4 +34,44 @@ TableToolbar.propTypes = {
    * Function used to translate parent table element in fullscreen
    */
   toggleFullscreen: PropTypes.func.isRequired,
+  /*
+   * Boolean value to know if there is some data fully loaded
+   */
+  isReady: PropTypes.bool.isRequired,
+  /*
+   * Boolean use to know if the user have rights to edit the solution
```

Comment:
Wrong description because `editMode` isn't exclusively defined by users rights, scenario parameters are disabled if the scenario has a validation status or during a run